### ISSUE TITLE
Erroring under windows-latest

### ISSUE THREAD

**Issue by mrecachinas** [jetli/wasm-pack-action#2]
Works perfectly (and efficiently) under macOS-latest and ubuntu-latest, but under windows-latest, it's throwing the following error:

```
Run jetli/wasm-pack-action@v0.2.0
Installing wasm-pack v0.9.1 ...
Downloading wasm-pack from https://github.com/rustwasm/wasm-pack/releases/download/v0.9.1/wasm-pack-v0.9.1-x86_64-pc-windows-msvc.tar.gz ...
Extracting wasm-pack to C:\Users\RUNNER~1\AppData\Local\Temp\setup-wasm-pack ...
C:\windows\system32\tar.exe xz -C C:\Users\RUNNER~1\AppData\Local\Temp\setup-wasm-pack -f D:\a\_temp\6dc27533-e606-4b47-baee-68dde941d860
##[error]ENOENT: no such file or directory, rename 'C:\Users\RUNNER~1\AppData\Local\Temp\setup-wasm-pack\wasm-pack.exe' -> 'C:\Users\runneradmin\.cargo\bin\wasm-pack.exe'
Done
```

Do you have any suggestions on how to fix this? I would have thought your

```js
await io.mkdirP(execFolder)
```

on [main.ts#L64](https://github.com/jetli/wasm-pack-action/blob/master/src/main.ts#L64) would be sufficient, but it appears the 

```js
await io.mv(path.join(extractedFolder, archTopFolder, exec), execPath)
```

is still failing.

**Comment by jetli** [comment]
Fixed and released v0.3.0, please try again.

**Comment by mrecachinas** [comment]
That did the trick -- thanks!
